cardano-cli/src/Cardano/CLI/Run/Friendly.hs

```diff
@@ -11,167 +11,203 @@
 module Cardano.CLI.Run.Friendly (friendlyTxBodyBS) where
 
 import           Cardano.Prelude
+import qualified Prelude
 
-import           Data.Aeson (Object, Value (..), object, toJSON, (.=))
-import qualified Data.HashMap.Strict as HashMap
+import           Data.Aeson (Value (..), object, toJSON, (.=))
+import           Data.Yaml (array)
 import           Data.Yaml.Pretty (defConfig, encodePretty, setConfCompare)
 
 import           Cardano.Api as Api (AddressInEra (..),
-                   AddressTypeInEra (ByronAddressInAnyEra, ShelleyAddressInEra), CardanoEra,
-                   ShelleyBasedEra (ShelleyBasedEraAllegra, ShelleyBasedEraMary, ShelleyBasedEraShelley),
-                   ShelleyEra, TxBody, serialiseAddress)
-import           Cardano.Api.Byron (TxBody (ByronTxBody))
-import           Cardano.Api.Shelley (TxBody (ShelleyTxBody), fromShelleyAddr)
-import           Cardano.Binary (Annotated)
-import qualified Cardano.Chain.UTxO as Byron
-import           Cardano.Ledger.Shelley as Ledger (ShelleyEra)
-import           Cardano.Ledger.ShelleyMA (MaryOrAllegra (Allegra, Mary), ShelleyMAEra)
-import qualified Cardano.Ledger.ShelleyMA.TxBody as ShelleyMA
-import           Ouroboros.Consensus.Shelley.Protocol.Crypto (StandardCrypto)
-import           Shelley.Spec.Ledger.API (Addr (..), TxOut (TxOut))
+                   AddressTypeInEra (ByronAddressInAnyEra, ShelleyAddressInEra),
```

Comment:
We should arrange the imports such that you can just have `import Cardano.Api` otherwise the imports will continue to get even more verbose.

Reply:
Is implicit import list ok here?

Reply:
Changed to implicit import